includes/variables.php

```diff
@@ -753,14 +744,10 @@ function tsml_load_config()
                 'O' => __('Open', '12-step-meeting-list'),
                 'OUT' => __('Outdoor Meeting', '12-step-meeting-list'),
                 'POC' => __('People of Color', '12-step-meeting-list'),
-                'POL' => __('Polish', '12-step-meeting-list'),
-                'POR' => __('Portuguese', '12-step-meeting-list'),
                 'P' => __('Professionals', '12-step-meeting-list'),
-                'PUN' => __('Punjabi', '12-step-meeting-list'),
-                'RUS' => __('Russian', '12-step-meeting-list'),
                 'A' => __('Secular', '12-step-meeting-list'),
                 'SEN' => __('Seniors', '12-step-meeting-list'),
-                'ASL' => __('Sign Language', '12-step-meeting-list'),
+				'ASL' => __('American Sign Language', '12-step-meeting-list'),
```

Comment:
we do want to change this at some point, but i wouldn't make this change as part of this PR